packages/components/src/date-time/date-time/index.tsx

```diff
@@ -31,6 +32,8 @@ function UnforwardedDateTimePicker(
 		onMonthPreviewed = noop,
 		onChange,
 		events,
+		__nextRemoveHelpButton = false,
+		__nextRemoveResetButton = false,
```

Comment:
We should probably add a deprecation warning for both props ([example](https://github.com/WordPress/gutenberg/blob/6eff9a514591f3b14f667e94b745eaaa507d912a/packages/components/src/card/card/hook.js#L26-L29))